frontend/src/components/App/index.tsx

```diff
@@ -1,8 +1,7 @@
 import React, { Component } from 'react';
 import styled from 'styled-components';
 import fetchHello from '../../api/hello';
-import LandingPage from '../../views/Landing';
-import logo from '../../logo.svg';
+import Editor from "@monaco-editor/react";
```

Comment:
nit: Reorder above relative imports (e.g. '../../logo.svg') and alphabetically by package within package imports (e.g. 'react').